cirq-google/cirq_google/serialization/common_serializers.py

```diff
@@ -508,72 +500,30 @@ def _add_phase_match(op: cirq.Operation, proto: v2.program_pb2.Operation):
     ),
 ]
 
-
-#
-# FSim serializer
-# Only allows iswap, sqrt_iswap and their inverses, iswap, CZ, identity, and sycamore
-# Note that not all combinations may not be available on all processors
-def _can_serialize_limited_fsim(theta: float, phi: float):
-    # Symbols for LIMITED_FSIM are allowed, but may fail server-side
-    # if an incorrect run context is specified
-    if _near_mod_2pi(phi, 0) or isinstance(phi, sympy.Symbol):
-        if isinstance(theta, sympy.Symbol):
-            return True
-        # Identity
-        if _near_mod_2pi(theta, 0):
-            return True
-        # sqrt ISWAP
-        if _near_mod_2pi(theta, -np.pi / 4):
-            return True
-        # inverse sqrt ISWAP
-        if _near_mod_2pi(theta, np.pi / 4):
-            return True
-        # ISWAP
-        if _near_mod_2pi(theta, -np.pi / 2):
-            return True
-        # Inverse ISWAP
-        if _near_mod_2pi(theta, np.pi / 2):
-            return True
-    # Sycamore
-    if (
-        (_near_mod_2pi(theta, np.pi / 2) or isinstance(theta, sympy.Symbol))
-        and (_near_mod_2pi(phi, np.pi / 6))
-        or isinstance(phi, sympy.Symbol)
-    ):
-        return True
-    # CZ
-    if (
-        (_near_mod_2pi(theta, 0) or isinstance(theta, sympy.Symbol))
-        and (_near_mod_2pi(phi, np.pi))
-        or isinstance(phi, sympy.Symbol)
-    ):
-        return True
-    return False
-
-
-def _can_serialize_limited_iswap(exponent: float):
-    # Symbols for LIMITED_FSIM are allowed, but may fail server-side
-    # if an incorrect run context is specified
-    if isinstance(exponent, sympy.Symbol):
-        return True
-    # Sqrt ISWAP
-    if _near_mod_n(exponent, 0.5, 4):
-        return True
-    # Inverse Sqrt ISWAP
-    if _near_mod_n(exponent, -0.5, 4):
-        return True
-    # ISWAP
-    if _near_mod_n(exponent, -1.0, 4):
-        return True
-    # Inverse ISWAP
-    if _near_mod_n(exponent, 1.0, 4):
-        return True
-    # Identity
-    if _near_mod_n(exponent, 0.0, 4):
-        return True
-    return False
-
-
+_LIMITED_FSIM_GATE_FAMILY = common_gate_families.FSimGateFamily(
+    gates_to_accept=[
+        cirq.IdentityGate(2),
+        cirq.SQRT_ISWAP_INV,
+        cirq.SQRT_ISWAP,
+        cirq.ISWAP,
+        cirq.ISWAP ** -1,  # type: ignore
```

Comment:
We might want to define `cirq.ISWAP_INV` by analogy with `cirq.SQRT_ISWAP_INV`.

Reply:
This can be an independent change. Created https://github.com/quantumlib/Cirq/issues/4615 to track this.